carrierwave.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |s|
   s.add_dependency "activesupport", ">= 6.0.0"
   s.add_dependency "activemodel", ">= 6.0.0"
   s.add_dependency "image_processing", "~> 1.1"
-  s.add_dependency "marcel", "~> 1.0.0"
+  s.add_dependency "marcel", "~> 1.0.3"
```

Comment:
With the new code change, you don't need to make this version change, because the code is supported as far back as version 1.0.0.

Reply:
Reverted this change 👍🏻